idea/src/org/jetbrains/kotlin/idea/inspections/collections/ReplaceUselessCallWithToListFix.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.inspections.collections
+
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.openapi.project.Project
+import org.jetbrains.kotlin.idea.KotlinBundle
+import org.jetbrains.kotlin.psi.KtPsiFactory
+import org.jetbrains.kotlin.psi.KtQualifiedExpression
+import org.jetbrains.kotlin.psi.createExpressionByPattern
+
+class ReplaceUselessCallWithToListFix : LocalQuickFix {
```

Comment:
I'd generalize this quick fix and call it `ReplaceSelectorOfQualifiedExpressionFix` and put it to `org.jetbrains.kotlin.idea.quickfix` package. Then first line of class would look like:
```
class ReplaceSelectorOfQualifiedExpressionFix(val newSelector: String) : LocalQuickFix
```
I think you get the idea